rdwatch/core/views/model_run.py

```diff
@@ -315,6 +315,18 @@ def paginate_queryset(
                 )
 
             model_runs = qs | queryset.aggregate(**aggregate_kwargs)
+            if filters.region:
+                if qs['count'] == 0:  # No model runs we set bbox to Region bbox
+                    region_filter = filters.dict()['region']
+                    regions = [
+                        obj
+                        for obj in Region.objects.all()
+                        if obj.value == region_filter
+                    ]
+                    if len(regions) > 0:
+                        bbox = regions[0].geojson
+                        model_runs['bbox'] = bbox
```

Comment:
We can do this in pure SQL instead of python:
```suggestion
                    regions = Region.objects.filter(value=region_filter)
                    if regions.exists():
                        bbox = regions[0].geojson
                        model_runs['bbox'] = bbox
```

Reply:
Ahh I merged this without thinking, value is a property that I generate on the Region Model.  We needed some sort of unique value for Regions as a combination of the Name and the owner.  I tried the same thing initially.  I.E You and I can both have a Region called 'Custom' but we can't have two regions called 'Custom'.  So `value` is a simple concating of `${name}_${owner username} that way it works with our current dropdowns and the URL parameters.

We could swap all of our references to utilize Region Ids (ints) instead of the name.  It just means some more work on the front-end because I would like to keep the name as a URL query parameter like 'rgd.kitware.com/#/KR_R001`.  So I would have to utilize a name to ID mapping on the front end for the request.  Let me know if you want me to do that in this PR or if you need more clarification.

Reply:
We can postpone this for the future, I didn't realize at the time that `value` was a property.